src/debezium-service/templates/deployment.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Values.name}}
+  labels:
+    app: {{.Values.name}}
+  namespace: {{.Values.namespace.name}}
+spec:
+  progressDeadlineSeconds: 300
+  replicas: {{int .Values.instances}}
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: {{.Values.name}}
+  template:
+    metadata:
+      labels:
+        app: {{.Values.name}}
+        harness-team: PIE
+        harness-module: Platform
+      annotations:
+        cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: '8889'
+        prometheus.io/path: '/metrics'
+      {{- if .Values.istioInjectionEnabled}}
```

Comment:
this annotation is not required, instead can you follow the standards from other charts to add commonAnnotations and commonLabels, @avaakash can you review